apiaudio/helper_classes.py

```diff
@@ -74,3 +76,32 @@ def download(
             local_filenames.append(local_filename)
 
         return local_filenames
+
+
+class UploadableResource(APIRequest):
```

Comment:
this helper doesnt handle any exceptions, can not give proper error messages. please improve it